### ISSUE TITLE
[Tab/API] Tabs Should Not Encode Params, API should add setting encodeParameters

### ISSUE THREAD

**Issue by BigGrecian** [Semantic-Org/Semantic-UI#2752]
I am using ajax tabs to open up a customer's details and as such when the page is sent to the browser the tab url looks like so :
```
 <a class="active item" data-tab="profile.php?student=2027">Basic Profile</a> 

    <div class="ui bottom attached active tab segment" data-tab="profile.php?student=2027">
        
    </div>


```
However on upgrade to 2.0.7 the URL gets encoded as
./useredit/profile.php%3Fstudent%3D2027 

which obviously returns a 404 code as it expects?
.useredit/profile.php?student=2027

As below :

![image](https://cloud.githubusercontent.com/assets/12722365/9007676/a376d77a-3761-11e5-928a-81db42a9e549.png)

This used to work fine in 2.0.3 and before

Any idea how to rectify this?

**Comment by jlukic** [comment]
#2394 `urlData` is now `urlEncoded` by default. 

I can understand why this would cause issues particularly with `data-tab` which sometimes will include the query string as part of the parameter. 

It was a mistake not to include a setting to disable this. I'll add an `apiSetting` `encodeData` and make it `false` for tab.

**Comment by jlukic** [comment]
Fixed in `2.1`
